cicd/gitops/configuring-sso-on-argo-cd-using-dex.adoc

```diff
@@ -0,0 +1,23 @@
+[id="configuring-sso-for-argo-cd-using-dex"]
+= Configuring SSO for Argo CD using Dex
+include::modules/common-attributes.adoc[]
+include::modules/gitops-document-attributes.adoc[]
+:context: configuring-sso-for-argo-cd-using-dex
+
+toc::[]
+
+After the {gitops-title} Operator is installed, Argo CD automatically creates a user with `admin` permissions. To manage multiple users, Argo CD allows cluster administrators to configure SSO.
```

Comment:
```suggestion
After the {gitops-title} Operator is installed, Argo CD automatically creates a user with `admin` permissions. To manage multiple users, cluster administrators can use Argo CD to configure Single Sign-On (SSO).
```
Per IBM SG - See previous comment about avoiding statements in which inanimate objects grant abilities to people.